manifests/repo.pp

```diff
@@ -32,15 +32,14 @@
 
   case $::osfamily {
     'Debian': {
-      if !defined(Class['apt']) {
-        class { 'apt': }
-      }
+      include ::apt
+      Class['apt::update'] -> Package<| |>
```

Comment:
this should already be handled by the apt module it self, also when defining this relation multiple times with other modules might cause problems ?